reference-service/src/main/java/com/transformuk/hee/tis/reference/service/model/LocalOffice.java

```diff
@@ -28,6 +29,7 @@ public class LocalOffice implements Serializable {
   private Long id;
 
   @Column(name = "uuid")
+  @Type(type = "org.hibernate.type.UUIDCharType")
```

Comment:
I remember last time when I tried reading the type as UUIDCharType for some special UUID on Prod (eg. `0ee928ae-3efb-11`) I got an error. Have you also tried them too?

Reply:
I haven't, do you know how we've ended up with invalid UUIDs in the first place?
I could manually fix the problematic LO record 🤷🏻

Reply:
No idea and these issues exist in lots of reference tables.

Reply:
It does error if a contact is linked to that LO, in theory that should never happen since it's a "fake" LO anyway.
What did you do in the past, regen the UUID?

Reply:
I noticed this issue when adding a new reference type `fudningSubtype` a month ago. I don't think we have done anything yet to those existing ones as we didn't reference those UUIDs in the past. 
And, I think it's okay to regen the UUID, but don't forget NIMDTA.

Reply:
The UUIDs get corrupted whenever an update is made to the reference data, the new contact reference types should be fine but there's obviously some work needed to fix up the currently broken data and ensure that it can't happen.